src/packages/recompose/utils/mapValues.js

```diff
@@ -1,12 +1,14 @@
 const mapValues = (obj, func) => {
   const result = {}
-  /* eslint-disable no-restricted-syntax */
+  // TODO: no-unused-vars seems to trigger a issue in recent eslint versions
+  /* eslint-disable no-restricted-syntax, no-unused-vars */
   for (const key in obj) {
     if (obj.hasOwnProperty(key)) {
       result[key] = func(obj[key], key)
```

Comment:
I would like to find a nicer, cleaner solution for this without potentially breaking ES5 consumers.